<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UPWORKBOT</class>
 <widget class="QMainWindow" name="UPWORKBOT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>UPWORK</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>ACCOUNT 1</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>2</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>ENTER EMAIL ADDRESS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_1">
               <property name="text">
                <string>Config Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_2">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>SignUP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_10">
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_7">
               <property name="text">
                <string>AutoBID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_13">
               <property name="text">
                <string>10</string>
               </property>
               <property name="placeholderText">
                <string>Time(min)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_13">
               <property name="text">
                <string>STOP</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QSplitter" name="splitter_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_2">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>EMAIL ADDRESS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_3">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Full Name</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QGroupBox" name="groupBox_8">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Country</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_4">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Country</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>ACCOUNT 2</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>2</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>ENTER EMAIL ADDRESS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_3">
               <property name="text">
                <string>Config Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_4">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>SignUP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_11">
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_8">
               <property name="text">
                <string>AutoBID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_14">
               <property name="text">
                <string>10</string>
               </property>
               <property name="placeholderText">
                <string>Time(min)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_14">
               <property name="text">
                <string>STOP</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QSplitter" name="splitter_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QGroupBox" name="groupBox_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_6">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>EMAIL ADDRESS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QGroupBox" name="groupBox_11">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_7">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Full Name</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QGroupBox" name="groupBox_12">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Country</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_8">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Country</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_13">
          <property name="title">
           <string>ACCOUNT 3</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="lineEdit_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>2</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>ENTER EMAIL ADDRESS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_5">
               <property name="text">
                <string>Config Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_6">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>SignUP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_12">
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_9">
               <property name="text">
                <string>AutoBID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_15">
               <property name="text">
                <string>10</string>
               </property>
               <property name="placeholderText">
                <string>Time(min)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_15">
               <property name="text">
                <string>STOP</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QSplitter" name="splitter_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QGroupBox" name="groupBox_14">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="0" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_10">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>EMAIL ADDRESS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QGroupBox" name="groupBox_15">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_13">
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_11">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Full Name</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QGroupBox" name="groupBox_16">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_14">
               <item row="0" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Country</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_12">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Country</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
